src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  function getFormatDeclaration(prev, prop) {
+    return ({
+      ...prev,
+      [getTrim(prop[0])]: getTrim(prop[1]),
+    });
```

Comment:
redundant parentheses 
```suggestion
    return {
      ...prev,
      [getTrim(prop[0])]: getTrim(prop[1]),
    };
```